src/components/TextInput/TextInput.tsx

```diff
@@ -303,7 +303,9 @@ class TextInput extends React.Component<TextInputProps, State> {
         ios: false,
         default: true,
       }),
-    }).start(this.hidePlaceholder);
+    }).start(({ finished }) => {
+      if (finished) this.showPlaceholder;
```

Comment:
```suggestion
      if (finished) this.showPlaceholder();
```

Shouldn't this actually call the function?

Reply:
Ops! Good catch, thanks!